SickSangHae/View/MainView.swift

```diff
@@ -77,18 +76,53 @@ struct SearchBar: View {
     @FocusState var isInputActive: Bool
 
     var body: some View {
-        TextField("Enter your name", text: $text)
-            .textFieldStyle(.roundedBorder)
-                            .focused($isInputActive)
-                            .toolbar {
-                                ToolbarItemGroup(placement: .keyboard) {
-                                    Spacer()
-
-                                    Button("Done") {
-                                        isInputActive = false
-                                    }
-                                }
+        HStack {
+            Spacer()
+                .frame(width: 12)
+            
+            Image(systemName: "magnifyingglass")
+                .foregroundColor(Color("Gray200"))
+            
+            ZStack(alignment: .leading) {
+                if text.isEmpty {
+                    Text("식재료 검색")
+                        .foregroundColor(Color("Gray200"))
+                } else {
+                    EmptyView()
+                }
+                TextField("", text: $text)
+                    .focused($isInputActive)
+                    .toolbar {
+                        ToolbarItemGroup(placement: .keyboard) {
+                            Spacer()
+                            
+                            Button("완료") {
+                                isInputActive = false
                             }
+                        }
+                    }
+                
+                if !text.isEmpty {
+                    Button(action: {
+                        self.text = ""
+                    }) {
+                        HStack {
+                            Spacer()
+                            
+                            Image(systemName: "xmark.circle.fill")
+                                .foregroundColor(Color("Gray200"))
+                        }
+                    }
+                } else {
+                    EmptyView()
+                }
```

Comment:
```suggestion
                if text.isEmpty {
                    EmptyView()
                } else {
                    Button(action: {
                        self.text = ""
                    }) {
                        HStack {
                            Spacer()
                            
                            Image(systemName: "xmark.circle.fill")
                                .foregroundColor(Color("Gray200"))
                        }
                    }
                }
```

읽기 쉽게 작성될 수 있을거같아요 ~~~ ! : )